pkg/tests/testinfra/testinfra.go

```diff
@@ -202,6 +202,13 @@ func CreateGrafDir(t *testing.T, opts ...GrafanaOpts) (string, string) {
 	_, err = anonSect.NewKey("enabled", "true")
 	require.NoError(t, err)
 
+	alertingSect, err := cfg.NewSection("alerting")
+	require.NoError(t, err)
+	_, err = alertingSect.NewKey("notification_timeout_seconds", "1")
+	require.NoError(t, err)
+	_, err = alertingSect.NewKey("evaluation_timeout_seconds", "1")
```

Comment:
I don't think we use this configuration in `ngalert` evaluations yet. This should be part of grafana/alerting-squad#46